### DISCUSSION TITLE
DEVC position

### DISCUSSION THREAD

**Discussion by Nadezhda-Kontar** [firemodels/fds#13033] [Q&A]
Hello! Can you help me please.
I need measure temperature on Z=1.7 with DEVC, Mesh cell is 0.25. 
In User Guide I see that DEVC reports the value in the cell containing the point XYZ and temperature is defined at cell center. 
Does it mean that I actualy have results not for Z=1.7 but for Z=(1.5+1.75)/2=1.625 (cell center Z)? If so, has FDS some interpolation function so I can get results from DEVC position exactly?

**Comment by rmcdermo** [comment]
Search the user guide for SPATIAL_STATISTIC='INTERPOLATION'.
